tests/phpunit/payment-methods/class-wc-stripe-express-checkout-ajax-handler-test.php

```
@@ -98,4 +100,103 @@ public function provide_test_modify_country_locale_for_express_checkout() {
+	 * @preserveGlobalState disabled
+	 */
+	public function test_ajax_add_to_cart_returns_success_for_simple_product() {
+		Ajax_Test_Helper::init_hooks();
+
+		$product = WC_Helper_Product::create_simple_product();
+
+		$this->express_checkout_helper->expects( $this->once() )
+			->method( 'supported_product_types' )
+			->willReturn( [ ProductType::SIMPLE ] );
+
+		$display_items = [
```

PHPMD (2.15.0)

[error] The variable $display_items is not named in camelCase. (undefined)

(CamelCaseVariableName)